Siri/L5/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
```

Comment:
Good job adding comments to each method giving a brief explanation. However, perhaps adding Javadoc might be better!